consensus/istanbul/core/handler.go

```diff
@@ -156,7 +156,17 @@ func (c *core) handleMsg(payload []byte) error {
 	msg := new(message)
 	if err := msg.FromPayload(payload, c.validateFn); err != nil {
 		if c.backend.NodeType() == common.CONSENSUSNODE {
-			logger.Error("Failed to decode message from payload", "err", err)
+			if err != istanbul.ErrUnauthorizedAddress {
+				logger.Error("Failed to decode message from payload", "err", err)
+			} else {
```

Comment:
I think returning err early makes the code more clean and intuitive.

if err != istanbul.ErrUnauthorizedAddress {
    ...
    return err
}

// do the rest of the code